packages/react-integration/cypress/integration/labelgroup.spec.ts

```diff
@@ -0,0 +1,28 @@
+describe('Chip Group Demo Test', () => {
```

Comment:
there's a bunch of stuff in here labeled 'chip' - should it be 'label'?

Reply:
Thanks @mcoker good catch - I've updated the integration tests for this component